addon/components/object-list-view.js

```diff
@@ -502,6 +502,10 @@ export default FlexberryBaseComponent.extend(
     }
 
     let cols = this._generateColumns(projection.attributes);
+    if ('caption' in projection) {  //GroupEdit has caption values, dont use userSettings, return basic order
```

Comment:
Can you add more detailed comment what condition "('caption' in projection)" does?